components/raftstore-v2/src/batch/store.rs

```diff
@@ -64,19 +64,17 @@ struct StorePoller<EK: KvEngine, T> {
 
 impl<EK: KvEngine, T> StorePoller<EK, T> {
     pub fn new(poll_ctx: StoreContext<T>, cfg_tracker: Tracker<Config>) -> Self {
-        let mut poller = Self {
+        StorePoller {
```

Comment:
I use `Self` on purpose. I have found it very convenient when the name is changed several times during development. Though I'm OK with either way.